claragenomics/dl4atac/layers.py

```diff
@@ -15,17 +15,17 @@
 
 
 class ZeroSamePad1d(nn.Module):
-    """describe."""
+    """Apply same zero padding to input."""
```

Comment:
Same zero padding as what ?

Reply:
`same` is a type of padding that allows output size to be kept same as input

https://www.quora.com/What-does-the-same-padding-parameter-in-convolution-mean-in-TensorFlow